LandSandBoat/server
# [trust] Add and hook up TRUST_DISTANCE mobmod

**_I affirm:_**

## What does this pull request do?

Refactors trust movement settings to be a mobMod instead of piggybacking on the mob_pools.behavior field.

## Steps to test these changes

Summon Shantotto, Joachim, and Semih-Lafinah. !fafnir. Start fighting. See them reposition as they're meant to - with no errors in the logs etc.
